jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java

```diff
@@ -96,8 +97,16 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             getSession(),
             getLog(),
             tempDirectoryProvider);
+
+    GlobalConfig globalConfig = null;
+    try {
+      globalConfig = GlobalConfig.readConfig();
+    } catch (IOException ex) {
+      throw new MojoExecutionException(ex.getMessage(), ex);
+    }
```

Comment:
I find it verbose too. I thought I would be able to clean this up on top of this change by further refactoring `UpdateChecker`, but I am facing trouble and not making progress.

The ugliness is due to initializing `updateCheckFuture` variable and having to use it in the `finally` block. We can put `GlobalConfig.readConfig()` in the `try` block (which already handles `IOException`), but then the `finally` block should check if `updateCheckFuture` is null.

Since I am not making progress on cleaning up this, I'll think of a few alternatives.